firefox-ios/Client/Frontend/Browser/Tabs/Views/RemoteTabsEmptyView.swift

```diff
@@ -58,16 +58,12 @@ class RemoteTabsEmptyView: UIView, ThemeApplicable {
         label.textAlignment = .center
     }
 
-    private let signInButton: LegacyResizableButton = .build { button in
-        button.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .callout,
-                                                                         size: UX.buttonSizeFont)
-        button.setTitle(.Settings.Sync.ButtonTitle, for: [])
-        button.layer.cornerRadius = UX.buttonCornerRadius
-        button.contentEdgeInsets = UIEdgeInsets(top: UX.buttonVerticalInset,
```

Comment:
given that you've removed this, if it's not used anywhere else in the file, this can be removed from the `UX` struct as well. Same for the other stuff you removed. :)

Reply:
I removed unused variables from the `UX` struct.